assets/install-rhcos.sh

```
@@ -1,53 +1,145 @@
+            --ignition-file "$IGNITION" \
+            --offline
+
+        if [ $? -ne 0 ]; then
+            ilog "ERROR: Failed to install Red Hat CoreOS."
+            exit 1
+        fi
+
+        sync
+
+        if efibootmgr -v | grep -q "Red-Hat CoreOS GRUB"; then
+            efibootmgr -b $(efibootmgr -v | grep "Red-Hat CoreOS GRUB" | awk '{print $1}' | cut -d' ' -f1) -B
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)